src/Controls/src/Core/Platform/AlertManager/PromptDialog.Windows.cs

```diff
@@ -56,8 +56,10 @@ void Initialize()
 		TextBlockMessage = new TextBlock { Text = "Message", TextWrapping = UI.Xaml.TextWrapping.Wrap };
 		TextBoxInput = new MauiPasswordTextBox();
 
+#pragma warning disable RS0030 // Do not use banned APIs; Panel.Children is banned for performance reasons. Here we allow it as a part of initialization.
 		layout.Children.Add(TextBlockMessage);
 		layout.Children.Add(TextBoxInput);
+#pragma warning restore RS0030 // Do not use banned APIs
```

Comment:
This one, you could have:
```suggestion
#pragma warning disable RS0030 // Do not use banned APIs; Panel.Children is banned for performance reasons. Here we can just cache it.
		var children = layout.Children;
#pragma warning restore RS0030 // Do not use banned APIs
		children.Add(TextBlockMessage);
		children.Add(TextBoxInput);
```